CONTRIBUTING.rst

```diff
@@ -167,3 +167,30 @@ To speed up the integration process, it helps if on GitHub you `allow
 maintainers to edit your pull request
 <https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/>`_
 so they can fix small issues autonomously.
+
+.. rubric: Review guidelines
+
+A pull request will probably be reviewed by someone in a different location,
+maybe even a different time zone. Therefore, avoiding blockings and
+misunderstandings is key. The following ideas could help ease and speed
+up the review process:
+
+- Be clear when something is ready to reviewed (see `WIP` above).
+- If a PR requires changes, say so explicitly, but politely (use emojis!).
+- When something is ready to be merge say it explicitly. If the reviewer does
+not have permission to merge, it should ping the corresponding person.
+- When reviewing, rate your comments. This would help the other person know If
+a comment must be addressed (e.g. it has substantial implications) or it is
+something the reviewer thought of but is a matter of "take it or leave it". A
+proposed scale is:
+  1. Comment/Shelved: Something worth mentioning, a doubt or question (comment).
+  If it will be treated later on as part of a different issue it is "shelved".
+  In case of shelving reference the issue.
+  2. Minor: Something that the reviewer thinks might need change. However, it
+  is not blocking, it is up to the developer to choose if and how to change
+  it. It can be merged!
+  3. Moderate: Something that requires further discussion (e.g. breaking
+  changes). It cannot be merged, unless explicitly said (e.g. choose a
+  solution proposed by the reviewer and implement it).
+  4. Major: Needs discussion, probably a video call. Even the opinion of an
```

Comment:
```suggestion
  4. Major: Needs more discussions, probably a chat. Even the opinion of an
```
or similar :)

Minor: what do you mean with the sentence "Even the opinion of an architect"? That you should involve one?

Reply:
Indeed, e.g. a change in `pidstore` base code or `invenio-records-rest` views.